vue/sbc-common-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sbc-common-components",
-  "version": "2.4.16",
+  "version": "2.4.17",
```

Comment:
Please also commit the package-lock.json file (so that the build server uses exactly the package versions that you used -- otherwise the build server will fetch the latest package versions, which may cause issues).

Reply:
Thanks. I have pushed the package-lock.json file as well